FoodFriends/app/src/main/java/com/example/foodfriends/adapters/ExploreAdapter.java

```diff
@@ -85,19 +94,24 @@ public ViewHolder(@NonNull View itemView){
             ivToGo = itemView.findViewById(R.id.ivExploreToGo);
         }
 
-        public void bind(Restaurant restaurant) {
-
+        /**
+         * Binds the xml elements to restaurant data
+         * */
+        public void bind(RestaurantObservable restaurant) {
+            restaurantObservable = restaurant;
+            restaurant.addObserver(this);
             tvRName.setText(restaurant.getName());
             tvAddress.setText(restaurant.getAddress());
-            num_likes =  restaurant.getLikes();
-            tvLikeCount.setText(String.valueOf(num_likes));
-            num_togo = restaurant.getToGos();
-            tvToGoCount.setText(String.valueOf(num_togo));
+            tvLikeCount.setText(String.valueOf(restaurant.getLikes()));
+            tvToGoCount.setText(String.valueOf(restaurant.getTogos()));
```

Comment:
Same comment about prefixes. What is tv?

Reply:
tv stands for TextView, which shows the type of view it is!

Reply:
ahhhh okay, I was like what is this? For what it's worth, general practice that i've seen is naming view objects something like likeCountTextView